service/controller/resource/tccp/create.go

```diff
@@ -121,6 +121,11 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 			if err != nil {
 				return microerror.Mask(err)
 			}
+		} else if key.IsAlreadyCreatedCluster(cr) {
```

Comment:
It will be unpredictable if we trigger a possible CF update every reconciliation loop, as mentioned below there could be  changes that are not applied yet and it would immediately trigger a change


Also  in the case of many cluster, we can easily hit the AWS API call limit if we try   an update every reconciliation loop

Reply:
but it is the reason why I implemented the AWS cache, to save API requests